source/elements/oneMKL/source/domains/rng/device_routines/device-distributions-template-parameter-method.rst

```diff
@@ -0,0 +1,49 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_uniform_continuous_distributions_method:
```

Comment:
I assume Poisson is not continuous one, maybe just:  _onemkl_device_rng_distributions_method:

Reply:
yes, missed this naming, thanks. changed